generate_openj9_scc.sh

```diff
@@ -0,0 +1,223 @@
+#!/bin/bash
+
+# Format taken from OpenLiberty SCC generation in docker images
+
+unset OPENJ9_JAVA_OPTIONS
+
+SCC_SIZE="50m" # Default size for SCC
+SCC_GEN_RUNS_COUNT=3 # Runs for generating SCC
+
+run_eclipse_and_stop() {
+
+    "$HOME"/eclipse-home/eclipse/eclipse &
+    ECLIPSE_PID=$!
+    
+    sleep 1m
+
+    kill -9 $ECLIPSE_PID
+    
+    sleep 10s
+
+}
+
+download_eclipse() {
+
+    # Creating a temporary directory for tomcat download
```

Comment:
Comment says tomcat, but you are downloading eclipse.

Reply:
Updated as eclipse